src/lib/constants.js

```diff
@@ -1,10 +1,10 @@
-export const CHANNEL_NAME = "Themer";
+export const CHANNEL_NAME = "FirefoxColor";
 
 export const DOWNLOAD_FIREFOX_URL =
-  "https://www.mozilla.org/firefox/new/?utm_campaign=themer-acquisition&utm_medium=referral&utm_source=themer.firefox.com";
+  "https://www.mozilla.org/firefox/new/?utm_campaign=firefoxcolor-acquisition&utm_medium=referral&utm_source=color.firefox.com";
 
 export const surveyUrl =
-  "https://qsurvey.mozilla.com/s3/Test-Pilot-Themer-Feedback";
+  "https://qsurvey.mozilla.com/s3/Test-Pilot-Firefox-Color-Feedback";
```

Comment:
This will break the survey link even more - we should fix it in issue #136 and either move the survey to a renamed URL (and finish it) or change this URL back to "Themer" (and also finish the survey)